### ISSUE TITLE
Problem with entity resolution when calling via ENSEPRO

### ISSUE THREAD

**Issue by DenisAraujo68** [Ensepro/ensepro-core#31]
For the question
- Quando a princesa Diana morreu?

When calling Spotlight via ENSEPRO, it refer "princesa Diana" as "diana_prince" (see log file below), but when calling Spotlight in the command line, we got other reference:
> curl http://172.17.0.6/rest/annotate  --data-urlencode "text=Quando a princesa Diana morreu?"    -H  "accept: application/json"   --data "confidence=0.3" --data "support=0"
```
{
    "@text": "Quando a princesa Diana morreu?",
    "@confidence": "0.3",
    "@support": "0",
    "@types": "",
    "@sparql": "",
    "@policy": "whitelist",
    "Resources": [
        {
            "@URI": "http://pt.dbpedia.org/resource/Diana,_Princesa_de_Gales",
            "@support": "395",
            "@types": "Http://xmlns.com/foaf/0.1, ...
            "@surfaceForm": "princesa Diana",
            "@offset": "9",
            "@similarityScore": "0.9999997433595854",
            "@percentageOfSecondRank": "0.0"
        }
    ]
}
```
Log file: [ensepro.log](https://github.com/Ensepro/core/files/2496660/ensepro.log)

**Comment by DenisAraujo68** [comment]
The problem occurs when using confidence=0.8.
@alencarrh, we must change the configuration file to call with only 0.3 confidence.
As I don't know how to assign a commit to the issue, I will not do the change.
But, I'll be glad doing if you teach me how to do this.

**Comment by alencarrh** [comment]
Ok, to link a commit to an issue, you need to use these keywords in the commit message: 
`KEYWORD #ISSUE_ID`

```
close
closes
closed
fix
fixes
fixed
resolve
resolves
resolved
```
more details can be found here: https://help.github.com/articles/closing-issues-using-keywords/

When you commit to **develop**, the issue will not close. 
> Once the branch is merged into the default branch, the issue will close.

Our default branch is **master**, so once the **develop** is merge into it, it will close the issue.

Suggestion of commit message:
`Updating spotlight confidences, closes #31`

**Comment by DenisAraujo68** [comment]
*DenisAraujo68* closed this issue in 7b9b37a
